genon/preprocessor/src/main.py

```
@@ -117,6 +117,40 @@ async def parse(
+    try:
+        logger.info('[chunker] Start')
+        # 앞단계(파싱) 결과 docling JSON 은 params["document"] 로 인라인 전달된다.
+        data = await processor(request, file_path, **params)
+        logger.info('[chunker] Success')
+    except GenosServiceException as e:
+        logger.error(f'[chunker] Error\n{traceback.format_exc()}\n')
+        return JSONResponse(
+            {'code': 1, 'errMsg': e.error_msg, 'data': None,
+             'error_code': e.error_code, 'error_msg': e.error_msg},
+            status_code=200)
+    except Exception as e:
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)